crates/bevy_render/src/render_resource/gpu_array_buffer.rs

```diff
@@ -24,13 +24,13 @@ impl<T: ShaderType + ShaderSize + WriteInto + Clone> GpuArrayBufferable for T {}
 /// binding (within reasonable limits).
 ///
 /// Other options for storing GPU-accessible data are:
-/// * [`StorageBuffer`](crate::render_resource::StorageBuffer)
+/// * [`BufferVec`](crate::render_resource::BufferVec)
```

Comment:
```suggestion
/// * [`BufferVec`]
```
You don't need to link to BufferVec's location explicitly - it's already in scope.

Reply:
Ya this was my mistake; I wasn't paying proper attention to the scope